apps/sms/views/inbox/js/inbox.js

```diff
@@ -146,7 +149,7 @@ var InboxView = {
     var draftId = node.dataset.draftId;
 
     var threadOrDraft = draftId ?
-      Drafts.byDraftId(+draftId) : Threads.get(threadId);
+      Drafts.byDraftId(+draftId) : Threads.get(+threadId);
```

Comment:
I'm fine for that if it save your time for writing the test since it's no harm for codebase.